src/ServiceInsight.Tests/packages.config

```diff
@@ -5,6 +5,7 @@
   <package id="Caliburn.Micro.Core" version="2.0.2" targetFramework="net45" />
   <package id="NSubstitute" version="1.10.0.0" targetFramework="net45" />
   <package id="NUnit" version="3.2.1" targetFramework="net45" />
+  <package id="ObservablePropertyChanged" version="0.1.3" targetFramework="net452" developmentDependency="true" />
```

Comment:
Is this effectively a transitive dependency? I.e. if we stopped using `ObservablePropertyChanged` in the `ServiceInsight` project, would we still use it in `ServiceInsight.Tests` for something else? If the answer is no, I think it's probably better to leave it as transitive and omit it from this packages.config.